charts/misc-apps/values.yaml

```diff
@@ -96,3 +96,22 @@ chartmuseum:
   # chartmuseum.values -- Helm values
   # @default -- [upstream values](https://github.com/chartmuseum/charts/blob/main/src/chartmuseum/values.yaml)
   values: {}
+
+# prom2teams -- [prom2teams](https://github.com/prometheus-msteams/prometheus-msteams) ([example](./example/prom2teams.yaml))
+# @default -- -
+prom2teams:
```

Comment:
how did we end up with this name? the project's name is prometheus-msteams

Reply:
We can always change it, I ended up using this as previously I was using this project instead https://github.com/idealista/prom2teams , but I am changing it then, thanks for the suggestion.